_toc.yml

```
@@ -573,37 +573,54 @@ entries:
           - file: docs/products/flink/reference/flink-limitations
             title: Limitations
       - file: docs/products/flink/getting-started
-        title: Get started
+        title: Quickstart
       - file: docs/products/flink/concepts
         title: Concepts
         entries:
-          - file: docs/products/flink/concepts/flink-for-analysts
-          - file: docs/products/flink/concepts/flink-for-operators
           - file: docs/products/flink/concepts/flink-applications
-            title: Aiven for Apache Flink applications
-          - file: docs/products/flink/concepts/kafka-connectors
-          - file: docs/products/flink/concepts/kafka-connector-requirements
+            title: Aiven Flink applications
           - file: docs/products/flink/concepts/supported-syntax-sql-editor
-          - file: docs/products/flink/concepts/event-processing-time
-          - file: docs/products/flink/concepts/windows
-          - file: docs/products/flink/concepts/watermarks
-          - file: docs/products/flink/concepts/checkpoints
           - file: docs/products/flink/concepts/tables
+            title: Flink tables
+          - file: docs/products/flink/concepts/checkpoints
           - file: docs/products/flink/concepts/savepoints
+          - file: docs/products/flink/concepts/event-processing-time
+          - file: docs/products/flink/concepts/watermarks
+          - file: docs/products/flink/concepts/windows
+          - file: docs/products/flink/concepts/kafka-connectors
+          - file: docs/products/flink/concepts/kafka-connector-requirements
+          
+          
+          
       - file: docs/products/flink/howto
         title: HowTo
         entries:
+          - file: docs/products/flink/howto/list-get-started
+            title: Get started
           - file: docs/products/flink/howto/create-integration
-          - file: docs/products/flink/howto/create-flink-applications
-          - file: docs/products/flink/howto/manage-flink-applications
-            title: Manage Aiven for Apache Flink applications
-          - file: docs/products/flink/howto/connect-kafka
-          - file: docs/products/flink/howto/connect-pg
-          - file: docs/products/flink/howto/connect-opensearch
-          - file: docs/products/flink/howto/create-job
+            title: Data service integrations
+          - file: docs/products/flink/howto/list-flink-applications
+            title: Aiven Flink applications
```

This might be a trademark violation if we leave it like this

```suggestion
            title: Aiven for Apache Flink applications
```

---

Fixed this.